content-src/components/NewTabPage/NewTabPage.js

```diff
@@ -4,22 +4,36 @@ const {connect} = require("react-redux");
 const TopSites = require("components/TopSites/TopSites");
 const ActivityFeed = require("components/ActivityFeed/ActivityFeed");
 const Spotlight = require("components/Spotlight/Spotlight");
+const Search = require("components/Search/Search");
 
 const NewTabPage = React.createClass({
+  onSearch(value) {
+    window.location = `http://search.yahoo.com/search?p=${encodeURIComponent(value)}`;
```

Comment:
+1 to HTTPS that URL!

(It looks like Yahoo! redirects to https://search.yahoo.com/search?p=pants anyways, but we could save at least one redirect. Plus, HTTPS is awesome.)

Not sure if we want to extract the search prefix into some config.yml or something, or if we can detect the user's preferred search provider and default to that.